addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/dialogs/ZestRecordScriptDialog.java

```diff
@@ -176,6 +186,35 @@ private List<String> getSites() {
         return list;
     }
 
+    private void launchBrowser(String url, String browserName) {
+        ExtensionSelenium extSelenium =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionSelenium.class);
+        try {
+            WebDriver wd = extSelenium.getProxiedBrowserByName(browserName);
+            if (wd == null) {
+                throw new NullPointerException();
+            }
+            String zapurl = wd.getCurrentUrl();
+            wd.get(url);
+            JavascriptExecutor jsExecutor = (JavascriptExecutor) wd;
+            jsExecutor.executeScript("localStorage.setItem('localzapurl', '" + zapurl + "')");
+            jsExecutor.executeScript("localStorage.setItem('localzapenable',false)");
+        } catch (NullPointerException e) {
+            String msg;
+            if (browserName == CHROME) {
+                msg = extSelenium.getWarnMessageFailedToStart(Browser.CHROME);
+            } else {
+                msg = extSelenium.getWarnMessageFailedToStart(Browser.FIREFOX);
+            }
+            cancelPressed();
+            LOGGER.error(msg);
```

Comment:
`warn` but since the message is being shown to the user I don't think it's necessary to log it (unless it's changed to include the exception).